packages/contracts-bedrock/deploy-config/goerli.json

```diff
@@ -1,26 +1,39 @@
 {
-  "l1StartingBlockTag": "0xafce66a0a2446856112e4069b275ad32b1f4a607888f9c4c59eddf9be81f8670",
+  "finalSystemOwner": "DUMMY",
+  "controller": "DUMMY",
+
+  "l1StartingBlockTag": "DUMMY",
   "l1ChainID": 5,
-  "l2ChainID": 111,
+  "l2ChainID": 420,
   "l2BlockTime": 2,
 
-  "maxSequencerDrift": 1000,
-  "sequencerWindowSize": 120,
+  "maxSequencerDrift": 3600,
+  "sequencerWindowSize": 7200,
   "channelTimeout": 120,
-  "p2pSequencerAddress": "0x9965507D1a55bcC2695C58ba16FB37d819B0A4dc",
-  "batchInboxAddress": "0xff00000000000000000000000000000000000002",
-  "batchSenderAddress": "0xa11d2b908470e17923fff184d48269bebbd9b2a5",
-
-  "systemConfigOwner": "0xa0Ee7A142d267C1f36714E4a8F75612F20a79720",
 
-  "l2OutputOracleSubmissionInterval": 6,
+  "p2pSequencerAddress": "DUMMY",
+  "batchInboxAddress": "0xff00000000000000000000000000000000000000",
+  "batchSenderAddress": "DUMMY",
+  "l2OutputOracleSubmissionInterval": 20,
   "l2OutputOracleStartingTimestamp": -1,
-  "l2OutputOracleProposer": "0x6c23a0dcdfc44b7a57bed148de598895e398d984",
-  "l2OutputOracleChallenger": "0x6925b8704ff96dee942623d6fb5e946ef5884b63",
+  "l2OutputOracleProposer": "DUMMY",
+  "l2OutputOracleChallenger": "DUMMY",
+  "finalizationPeriodSeconds": 2,
+
+  "systemConfigOwner": "DUMMY",
+
+  "l2GenesisBlockGasLimit": "0x1C9C380",
+  "l2GenesisBlockCoinbase": "0x42000000000000000000000000000000000000f0",
+  "l2GenesisBlockBaseFeePerGas": "0x3b9aca00",
```

Comment:
I am in favor of removing `l2GenesisBlockBaseFeePerGas` to simplify things, if the value is not sane then it will make eip1559 less stable after the migration block.

I propose we use this formula:

```
	gasUsed := (uint64)(config.L2GenesisBlockGasLimit) * config.EIP1559Elasticity
```

So that we hit this line here: https://github.com/ethereum-optimism/op-geth/blob/e383f9911586765c9792357da37a6c4020fcc7bf/consensus/misc/eip1559.go#L65

Reply:
Hmm, I'm on the fence about this since it only matters for networks that we're migrating. Concerned that adding the formula will introduce hidden complexity.

Reply:
What do you think?

Reply:
I have a preference for using `params.InitialBaseFee` as the initial base fee across our networks to remove additional config that could be a footgun as done in https://github.com/ethereum-optimism/optimism/pull/4139/files

Reply:
If you have a strong preference for manually configuring these values then we can do that, I have a preference to remove as much config as possible to reduce the chances of misconfiguration. I'd like the eip1559 stuff to be stable in the beginning

Reply:
Either way, we will need to do the eip 1559 math under the hood with these values to ensure that its not extremely volatile at the start of the network

Reply:
why is it not the params.InitialBasefee? That seems like it should work automatically

Reply:
Ok, I'll remove this configuration in a subsequent PR. That'll span multiple projects so I don't want to do it here.